src/main/java/org/apache/sysml/api/python/SystemML.py

```diff
@@ -247,3 +257,163 @@ def getStringRDD(self, varName, format):
         #    return rdd
         #except Py4JJavaError:
         #    traceback.print_exc()
+
+def getNumCols(numPyArr):
+	if len(numPyArr.shape) == 1:
```

Comment:
`np.ndim` is the preferred way to do this. Btw, in 1-D arrays, the number of elements are generally regarded as the number of features.

(Also in sklearn, we deprecated the use of 1-D arrays. All data should be provided as a 2-D array)